### ISSUE TITLE
Device not broadcasting after replaced battery?

### ISSUE THREAD

**Issue by YvoTuk** [atc1441/ATC_MiThermometer#223]
One of my ATC_Mi thermometers recently drained it's battery and got empty.
After replacing the battery, all seemed fine, but I noticed it seems to have stopped broadcasting.
I connected using TelinkFlasher which worked, tried some settings with no effect (the display changes when the setting are sent, but no data on the ESP32 data collector.
I then uploaded the original firmware to see if it was corrupted in some way. Although the upload is very slow (takes over 15mins!) it worked fine.
I activated the device in TelinkFlasher which returned a new Bindkey. Noted that and flashed the same firmware as I used for all the other devices.
Copied / pasted the Bindkey in the ESP32 file and .. nothing.

I don't understand what's happening or what's caused this to happen. What am I missing?

**Comment by devzwf** [comment]
experiencing about the same 
device seems stop broadcasting mean i cannot even see it to pair it anymore.
I have 2 device experiencing this .....
one the device still broadcast but i cannot stay connected enough to update the fw
is solder the pad the only way to fix this  ?

**Comment by atc1441** [comment]
If no Bluetooth is available anymore the only back debricking is via UART yes.

Would always try a new battery, even if the devices are new, first

**Comment by devzwf** [comment]
i already try new battery
device is working fine... just no Bluetooth
And i forgot to say this did not happen after battery dried.....
all suddenly the bluetooth stop working as i said , secodn device

**Comment by kadamski** [comment]
Strange. Did you try connecting to the device with some other computer or even a phone, maybe?

**Comment by devzwf** [comment]
Yes i try to pair from different device , but there is simply no advertising.
Like i said the device was sending data to esphome before and suddenly stop advertising

**Comment by Dogmanov** [comment]
Hey guys, after about a year of use, I started suffering similar with a limited number of my thermometers (4 of 21 units total. Maybe I'll spot a bigger pattern as more batteries star running low on juice...) I was able to reach them via bluetooth though and I eventually found this recovery procedure worked for me (I'm sure my process has room for improvement though.) Hopefully it'll help someone's woes...

1. Using the Telink flasher site and your mobile, connect to the misbehaving thermometer and reflash the stock firmware.
2. Reconnect to the device once it's flashed and grab the new bind key (in case you need it).
3. Before anything times out, reflash with the latest custom firmware.
4. Reconnect again once it's flashed and set the advertising type to 'mi like'.
5. At this point, I closed my mobile's BT connection, in case that was interfering with my BLE receivers, but the device seemed to work from thereon in.

**Comment by devzwf** [comment]
HI @Dogmanov 

I will revisit th eunit i have issue with... but my issue was i did not had any bluetooth at all...

But at one point , cant remember what i was doing i saw the BT sign on the device... but i still could not see it on the scanning device.... was weird... 
I then give up and use an other sensor.... and put this one i the box...

and this issue happen while i was at 60% battery...

**Comment by markvandenakker** [comment]
I have also this issue.
After replacing an empty battery, there is no data receiver from the device.
I've 8 devices, 6 of them are now offline....
I can reflash the device (flashed the latest firmware), and change setting via the website but nothing seems to help.
Anyone have a solution yet?

**Comment by atc1441** [comment]
It sounds line you need to select mi like advertising.

That depends on how you set them up first time

**Comment by atc1441** [comment]
It sounds line you need to select mi like advertising.

That depends on how you set them up first time

**Comment by bbogdanmircea** [comment]
Same issue here, can't see the sensors anymore on the esphome devices, also can't connect from the flashing page.
But I could see them from Windows BT Devices and even connect them and see the MAC address.
